src/main/java/org/jenkinsci/plugins/docker/workflow/client/WindowsDockerClient.java

```diff
@@ -46,7 +47,11 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
         }
         for (Map.Entry<String, String> variable : containerEnv.entrySet()) {
             argb.add("-e");
-            argb.addMasked(variable.getKey()+"="+variable.getValue());
+            if (variable.getKey().contains("") || variable.getValue().contains(" ")) {
```

Comment:
Right now this branch always runs, because `String.contains("")` is always true. I think you meant something like this:

```suggestion
            if (variable.getKey().contains("\"") || variable.getValue().contains(" ")) {
```

Reply:
It was intended to search for spaces in the the key, but as you pointed out this is already handled by the `WindowsUtil` function